### ISSUE TITLE
Can't get your exp-result

### ISSUE THREAD

**Issue by Eddieeee-Liu** [mit-han-lab/pvcnn#43]
Hi, sorry to bother, I got mIou 52% on S3DIS after running your code with default config. But It's still a big gap between ....

**Comment by zhijian-liu** [comment]
Could you please provide us with more details (which model you are running, whether you have changed any configurations)?

**Comment by Eddieeee-Liu** [comment]
Thx for your reply, I run with <python train.py configs/s3dis/pvcnn/area5/c1.py --devices 0,1> and my config is 

configs.seed = 1588147245
configs.deterministic = False

# data configs
configs.data = Config()
configs.data.num_workers = 16

configs.data.num_classes = 13

# dataset configs
configs.dataset = Config(S3DIS)
configs.dataset.root = '/home/share/Dataset/PointCloud/s3dis'
configs.dataset.with_normalized_coords = True
# configs.dataset.num_points = 2048
# configs.dataset.holdout_area = 5

# evaluate configs
configs.evaluate = Config()
configs.evaluate.fn = evaluate
configs.evaluate.num_votes = 1
configs.evaluate.batch_size = 10
configs.evaluate.dataset = Config(split='test')

# train configs
configs.train = Config()
configs.train.num_epochs = 50
configs.train.batch_size = 32
# save path
configs.train.save_path = "/home/jhliu/Research/Dev/exp-pc4"

# train: meters
configs.train.meters = Config()
configs.train.meters['acc/iou_{}'] = Config(MeterS3DIS, metric='iou', num_classes=configs.data.num_classes)
configs.train.meters['acc/acc_{}'] = Config(MeterS3DIS, metric='overall', num_classes=configs.data.num_classes)

# train: metric for save best checkpoint
configs.train.metric = 'acc/iou_test'

# train: criterion
configs.train.criterion = Config(nn.CrossEntropyLoss)

# train: optimizer
configs.train.optimizer = Config(optim.Adam)
configs.train.optimizer.lr = 1e-3

# model
configs.model = Config(PVCNN)
configs.model.num_classes = configs.data.num_classes
configs.model.extra_feature_channels = 6
configs.dataset.num_points = 4096

configs.train.optimizer.weight_decay = 1e-5
# train: scheduler
configs.train.scheduler = Config(optim.lr_scheduler.CosineAnnealingLR)
configs.train.scheduler.T_max = configs.train.num_epochs

configs.dataset.holdout_area = 5

**Comment by zhijian-liu** [comment]
@kentangSJTU, could you take a look at this? I'm not sure whether it is due to the version of PyTorch.

**Comment by jeromepatel** [comment]
Hello @Eddieeee-Liu, I am also experimenting with the model script but with another custom dataset. During my training using almost same hyper-parameters (except 4-5 variable with difference configuration), I have noticed significant variation when testing on same test dataset usiing same model when we keep `configs.evaluate.num_votes = 1`, I suspect that the inference is not same each time we run the script on the exact same trained model, so maybe you can try using higher value like `configs.evaluate.num_votes = 3 (or 5)` in your training or testing. Hope this helps, thanks.

**Comment by zhijian-liu** [comment]
Thanks @jeromepatel. @Eddieeee-Liu, I will close this issue for now, but feel free to reopen it if this issue has not been resolved.
